Source/JavaScriptCore/wasm/js/WebAssemblyExceptionConstructor.cpp

```diff
@@ -73,7 +73,10 @@ JSC_DEFINE_HOST_FUNCTION(constructJSWebAssemblyException, (JSGlobalObject* globa
     // Any GC'd values in here will be marked by the MarkedArugementBuffer until stored in the Exception.
     FixedVector<uint64_t> payload(values.size());
     for (unsigned i = 0; i < values.size(); ++i) {
-        payload[i] = toWebAssemblyValue(globalObject, tagFunctionType.argumentType(i), values.at(i));
+        auto type = tagFunctionType.argumentType(i);
+        if (UNLIKELY(type.kind == Wasm::TypeKind::V128 || isExnref(type)))
```

Comment:
We can use `type.isV128()` ?

Reply:
Either is fine, and I'm fine with `==  Wasm::TypeKind::V128`.